policy/modules/system/sysnetwork.if

```diff
@@ -454,6 +454,7 @@ interface(`sysnet_read_config',`
         init_search_pid_dirs($1)
 		allow $1 net_conf_t:dir list_dir_perms;
 		allow $1 net_conf_t:lnk_file read_lnk_file_perms;
+		allow $1 NetworkManager_var_run_t:dir list_dir_perms;
```

Comment:
"search_dir_perms" should be enough and is more appropriate.

I am really curious what that target of the /etc/resolv.conf symlink is labeled.
Can you do a `ls -al /etc/resolv.conf` , then do a `ls -alZ` on the target of the symlink and paste the results here please?